src/main/java/com/exadel/discountwebapp/vendor/entity/Vendor.java

```diff
@@ -7,11 +7,14 @@
 import org.springframework.data.jpa.domain.support.AuditingEntityListener;
 
 import javax.persistence.*;
+import javax.validation.constraints.Email;
 import java.time.LocalDateTime;
 
 @Entity
 @Data
-@Table(name = "vendor")
+@Table(name = "vendor", uniqueConstraints = {
+        @UniqueConstraint(columnNames = "vn_email")
```

Comment:
I'm not sure that we need this constraint declaration here, it should be defined in the liquibase script instead

Reply:
there is unique constraint in liquibase branch, so removed here